fennel/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## [0.18.10] - 2023-10-20
+- Add support for `starting_from` in S3 source.
```

Comment:
Suggest to call this `starting` instead of `starting_from`?

Reply:
Actually 'since' and 'until' may be even better? ('starting' is fine too but 'ending' doesn't roll off that well)